mw-webapp/src/component/wayCard/wayTag/WayTag.module.scss

```diff
@@ -0,0 +1,13 @@
+@import "src/styles/variables";
+
+$WayTagPadding: 4px 10px;
+$WayTagBorderRadius: 1000px;
```

Comment:
Why 1000px?  If you want to have circled border usually better to use 50%

Reply:
Value was taken from figma

Reply:
Value 50%  looks different , not like 1000px

Reply:
Do you think that 1000px better than 50%?

Reply:
If yes - merge